documentation/basics/headless.md

```diff
@@ -63,7 +63,10 @@ CPU and RAM resources can be controlled with environment variables or a config f
 **Optional input settings:**
 
 - `--readonly` Open the project in read-only mode; necessary for using a single project from multiple processes.
-- `--input_axes` specify meaning of your axes. In some cases ilastik's default guess for the axes is not correct. In such cases, supply the axes explicitly (e.g. `--input_axes=zcyx` for a 3D dataset with channels and the axes in the "zcyx" order).
+- `--input_axes` specify meaning of your axes. In some cases ilastik's default guess for the axes is not correct.
+  In such cases, supply the axes explicitly (e.g. `--input_axes=zcyx` for a 3D dataset with channels and the axes in the "zcyx" order).
```

Comment:
Maybe we should mention that for stack inputs, `--input-axes` refers to the axis order of each slice in the stack?

Reply:
added!